app/views/restaurants/_header.html.erb

```diff
@@ -0,0 +1,13 @@
+<div class="row">
+  <div class="col-lg-12">
+      <h1 class="page-header">
```

Comment:
A bit more semantic html would be nice, for example:

```html
<div class="row">
  <div class="col-lg-12">
      <h1 class="page-header">
        <%= @restaurant.name %>
      </h1>
      
      <section class="restaurant-details">
        <address>
          <%= @restaurant.address %>
        </address>
        <p class="pull-right">
          <%= @restaurant.price_range %>
        </p>
      </section>
  </div>
</div>
```

 and then we use the css to style it however we want